Common/Python/VolatilityModelPythonWrapper.cs

```diff
@@ -0,0 +1,79 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using Python.Runtime;
+using QuantConnect.Data;
+using QuantConnect.Securities;
+using System;
+using System.Collections.Generic;
+
+namespace QuantConnect.Python
+{
+    /// <summary>
+    /// Provides a volatility model that wraps a <see cref="PyObject"/> object that represents a model that computes the volatility of a security
+    /// </summary>
+    public class VolatilityModelPythonWrapper : IVolatilityModel
+    {
+        private readonly dynamic _model;
+
+        /// <summary>
+        /// Constructor for initialising the <see cref="VolatilityModelPythonWrapper"/> class with wrapped <see cref="PyObject"/> object
+        /// </summary>
+        /// <param name="model"> Represents a model that computes the volatility of a security</param>
+        public VolatilityModelPythonWrapper(PyObject model)
+        {
+            _model = model;
+        }
+
+        /// <summary>
+        /// Gets the volatility of the security as a percentage
+        /// </summary>
+        public decimal Volatility
+        {
+            get
+            {
+                return _model.Volatility;
```

Comment:
Out of curiosity, why don't we need the `using (Py.GIL()){ //... }` here?

Reply:
We don't need it when just reading the value of a member.